### DISCUSSION TITLE
return to original sender

### DISCUSSION THREAD

**Discussion by ugurtam** [postalserver/postal#2965] [Help with using Postal]
I need to remove the default return path of Postal, and (for example non-delivery emails) should return to original sender. Is this possible by making a change on the codes ?? I need this urgently ! Can anybody help ? I saw some discussions about this, but none of codes in these entries work.
@adamcooke  , it would be nice if you can help

**Comment by willpower232** [comment]
As Postal is the one that sent the email, it should know about what happened to the message so it can take whatever other steps it requires.

You could listen to the webhooks so you too know what happened to the message.

If you are routing regular email through Postal then you're probably having a bad time trying to use Postal in a way it was not made for.

**Reply by ugurtam** [reply]
I am using it as a smarthost service, and working without problems. I only want that Postal add to the header real sender address as return path. This is all. Users should get the out of office messages and non-delivery messages

**Comment by UEAB-BL** [comment]
I am also in need of this. In my case it is because of a bug on the receiving end and a custom mail filter they use. The bounce address is mistakenly being recognized as from address, causing it to not be deliverable through the filter.
I need to disable the return path bounce address until the other end can deploy their fix to this bug (its a process due it being quite large and governmental). Is there some way to disable it temporarily?

**Comment by blaueBrause** [comment]
I'd really love this feature too.
